docs/chainlink-nodes/configuration-variables.md

```diff
@@ -1560,6 +1562,15 @@ Example: `P2P_PEER_ID=12D3KooWMHMRLQkgPbFSYHwD3NBuwtS1AmxhvKVUrcfyaGDASR4U`
 
 ### Networking Stack V2
 
+> 🚧 If using the Networking Stack V2, you must unset the [Networking Stack V1](#networking-stack-v1) configuration variables:
+>
+> - [P2P_ANNOUNCE_IP](#p2p_announce_ip)
```

Comment:
I just want to be certain that we should unset all of these other vars. I know for a fact that the following should be unset:

```
P2P_LISTEN_PORT
P2P_ANNOUNCE_PORT
P2P_ANNOUNCE_IP
```

But not sure if `P2P_BOOTSTRAP_PEERS` and `P2P_PEER_ID` should be not be used if using v2 networking stack.

Reply:
Resolved based on @jmank88 's feedback.